src/megatron/bridge/models/qwen_vl/modelling_qwen3_vl/vision_model.py

```
@@ -0,0 +1,261 @@
+from dataclasses import dataclass
```

Flake8 (7.3.0)

[error] 'dataclasses.dataclass' imported but unused

(F401)